aip/0203.md

```diff
@@ -36,8 +36,8 @@ only. The purpose is to consistently document this behavior for users.
 
 ### Required.
 
-The use of `Required.` indicates that the field **must** be present on the
-request or resource.
+The use of `Required.` indicates that the field **must** be present (and set to
+a truthy value) on the request or resource.
```

Comment:
I don't think "truthy value" is a particularly useful way of expressing this. Non-empty value?

Reply:
Done.